### ISSUE TITLE
Alias not working

### ISSUE THREAD

**Issue by simonvomeyser** [steelbrain/node-ssh#122]
Hey @steelbrain , thanks a ton for this awesome package! 🎉 

It works perfectly, but I am having an Issue with using the alias command. Every combination I tried f.e. `execCommand("alias php='/usr/bin/php72'")` or other usage of the `alias` command seems to have no effect.

Is there any reason of the top of your head you can think of why this might be happening? Can you reproduce that? Even doing `source .bashrc` was not successful, all aliases I created with the user I log in with (even before that) are not working.

Would be awesome to get this to work since I use the package to execute a few commands that trigger further commands (like post_install_cmd scripts in a composer.json) that need to use the right version of php

Thanks so much in advance and keep up the good work 💪

**Comment by steelbrain** [comment]
Hi @simonvomeyser 

`execCommand` does not persist a shell session. If you want a proper shell session, one that remembers previous executions, try `requestShell()` instead

**Comment by simonvomeyser** [comment]
hey-ho @steelbrain :)

thanks for the quick reply, it led me on the right track to solve my problem. For anyone stumbling upon the same problem, I ended up with this:

```js
ssh.connect(credentials).then(function () {
  ssh.requestShell().then(function(stream) {

    // commands as proof of concept, alias php
    stream.write('alias test=php\n test -v\n'); 
    stream.end('\nexit\n');

    stream.on('data', function(data) {
      console.log(String(data)); // log data
    });
    stream.on('close', function(data) {
      ssh.dispose(); // close ssh connection
    });
  })
})
```

In this combination the `alias` command does indeed work. I'm a little shaky on the reasons why the command wasn't working before, even when used on it's own where it should should print all already defined aliases... but my SSH knowledge is not that thorough in the first place, so nevermind 🤓 

I took the code from code directly from the wrapped library [ssh-2](https://www.npmjs.com/package/ssh), to be precise from [here](https://www.npmjs.com/package/ssh2#start-an-interactive-shell-session)

I think this can be closed, thanks a lot for the help folks!  🤜 🤛

Simon vom Eyser
